### DISCUSSION TITLE
Publishing failed [Bug]

### DISCUSSION THREAD

**Discussion by bakastev** [webstudio-is/webstudio-community#190] [Q&A] [answered]
Every time i want to publish a Project (Copy/Paste from Webflow) to a subdomain i become this message: "Publish failed less than a minute ago". An earlier Publish of the same project was successful. I've made no big changes.

**Comment by bakastev** [comment]
https://p-61b898f4-0bc5-4ada-86d2-82eed40403dd.apps.webstudio.is/?authToken=6dfcd44b-c573-4953-83d8-97cd10e5b046&mode=edit

**Reply by kof** [reply, marked as answer]
We checked, somehow you got an empty attribute name on. the "BRIX - Logo Strip V10 Flex" instance.

We are investigating how that happened, but as a fix you can just delete this atribute

**Reply by bakastev** [reply]
Worked! Thank you. Do you have any best practices for copy/paste from webflow to be responsive? Unfortunately, not all viewpoints are displayed correctly after inserting.

**Reply by kof** [reply]
wdym by viewpoints?

some elements aren't supported https://docs.webstudio.is/university/foundations/copy-paste/webflow
